_upgrading/sopel-8.md

```diff
@@ -0,0 +1,420 @@
+such old plugins _actually still working_ is now very low indeed. Thus, we're
+not _dropping_ support for those decade-old plugins as much as we're _admitting_
+that _things have changed a lot_ and users would be better off seeking (or if
+necessary, writing) newer plugins with similar functionality.
+
+
+## Sopel 8 plugin changes
+
+### Removed built-in plugins
+
+Sopel's built-in plugins are slowly being migrated out to their own standalone
+packages, which will help the project manage responsibility for maintenance and
+updates in the long term. In 8.0, we say farewell to:
+
+* `help`: now published as [`sopel-help`][sopel-help]
+  * Note: Sopel still requires the `sopel-help` package, so it will be installed
+    and available automatically.
+* `ip`: now published as [`sopel-iplookup`][sopel-iplookup]
+* `meetbot`: now published as [`sopel-meetbot`][sopel-meetbot]
+* `py`: now published as [`sopel-py`][sopel-py]
+* `reddit`: now published as [`sopel-reddit`][sopel-reddit]
+* `remind`: now published as [`sopel-remind`][sopel-remind]
+
+  [sopel-help]: https://pypi.org/project/sopel-help/
+  [sopel-iplookup]: https://pypi.org/project/sopel-iplookup/
+  [sopel-meetbot]: https://pypi.org/project/sopel-meetbot/
+  [sopel-py]: https://pypi.org/project/sopel-py/
+  [sopel-reddit]: https://pypi.org/project/sopel-reddit/
+  [sopel-remind]: https://pypi.org/project/sopel-remind/
+
+### Notable changes to built-in plugins
+
+#### `currency` plugin
+
+The `fiat_provider` setting now takes precedence over the `fixer_io_key`.
+Previously, setting a `fixer_io_key` would use the `fixer.io` fiat exchange rate
+provider regardless of the `fiat_provider` setting.
+
+The old `fiat_provider` default of `exchangerate.host` is no longer a valid
+choice; it has been replaced by `open.er-api.com`.
+
+#### `reload` plugin
+
+The `.update` command has been removed from the `reload` plugin. Its functioning
+relied on running Sopel in a way that isn't officially supported, so we chose to
+remove it.
+
+#### `wikipedia` plugin
+
+The old command names (`.w`, `.wik`, and `.wiki`) are no longer used; they were
+freed up for use by bespoke plugins. `wikipedia` functionality is now invoked by
+the `.wikipedia` command, or `.wp` for short.
+
+The deprecated `lang_per_channel` config-file setting has been removed.
+
+
+## Sopel 8 API changes
+
+This is a good time to remind you that your plugins can specify both minimum and
+maximum Sopel versions in their requirements. If all plugins a bot owner wishes
+to install contain this metadata, they can ask `pip` to install Sopel and all of
+its plugins in a single command, and `pip`'s dependency solver will figure out
+which version of Sopel satisfies all of the plugins' needs.
+
+We do everything we can to keep breaking API changes to *only* major versions,
+so a version range like `sopel>=7.1,<9` is typically safe—at least the earliest
+Sopel version your plugin supports, up to (but excluding) the next unreleased
+major version.
```

Comment:
This goes with https://github.com/sopel-irc/sopel.chat/pull/46/files#r1548726609 — I needed the extra lines to make the suggestion work.

```suggestion
We do everything we can to keep breaking API changes to *only* major versions,
so a version range like `sopel>=8.0,<9` is typically safe. If you've tested
against older versions of Sopel, you can mark it as compatible with the earliest
tested version up to (but excluding) the next unreleased major version.
```